providers/cncf/kubernetes/provider.yaml

```diff
@@ -241,8 +241,8 @@ config:
         version_added: 8.1.0
         type: string
         example: ~
-        default: 'CreateContainerConfigError,ErrImagePull,CreateContainerError,ImageInspectError,
-        InvalidImageName'
+        default: "CreateContainerConfigError,ErrImagePull,CreateContainerError,ImageInspectError,\
+        InvalidImageName"
```

Comment:
This is the correct YAML syntax to ensure there is no extra space introduced in break-line